test/sparse/sparse.jl

```diff
@@ -1164,22 +1167,22 @@ end
     @test size(rotl90(a)) == (5,3)
 end
 
-@testset "Get Index Algs" begin
-    function test_getindex_algs{Tv,Ti}(A::SparseMatrixCSC{Tv,Ti}, I::AbstractVector, J::AbstractVector, alg::Int)
-        # Sorted vectors for indexing rows.
-        # Similar to getindex_general but without the transpose trick.
-        (m, n) = size(A)
-        !isempty(I) && ((I[1] < 1) || (I[end] > m)) && BoundsError()
-        if !isempty(J)
-            minj, maxj = extrema(J)
-            ((minj < 1) || (maxj > n)) && BoundsError()
-        end
-
-        (alg == 0) ? Base.SparseArrays.getindex_I_sorted_bsearch_A(A, I, J) :
-        (alg == 1) ? Base.SparseArrays.getindex_I_sorted_bsearch_I(A, I, J) :
-        Base.SparseArrays.getindex_I_sorted_linear(A, I, J)
+function test_getindex_algs{Tv,Ti}(A::SparseMatrixCSC{Tv,Ti}, I::AbstractVector, J::AbstractVector, alg::Int)
+    # Sorted vectors for indexing rows.
+    # Similar to getindex_general but without the transpose trick.
+    (m, n) = size(A)
+    !isempty(I) && ((I[1] < 1) || (I[end] > m)) && BoundsError()
+    if !isempty(J)
+        minj, maxj = extrema(J)
+        ((minj < 1) || (maxj > n)) && BoundsError()
     end
 
+    (alg == 0) ? Base.SparseArrays.getindex_I_sorted_bsearch_A(A, I, J) :
+    (alg == 1) ? Base.SparseArrays.getindex_I_sorted_bsearch_I(A, I, J) :
+    Base.SparseArrays.getindex_I_sorted_linear(A, I, J)
+end
+
+@testset "test_get_index_algs" begin
```

Comment:
I meant you should remove the underscore from get_index, not drop the test_ prefix